code/__HELPERS/time.dm

```diff
@@ -3,9 +3,11 @@
 #define MINUTES * 600
 #define HOURS   * 36000
 
-//Returns the world time in english
-proc/worldtime2text(timestamp = world.time)
-	return "[round(timestamp / 36000) + 12]:[(timestamp / 600 % 60) < 10 ? add_zero(timestamp / 600 % 60, 1) : timestamp / 600 % 60]"
+/proc/timestamp2text(timestamp = 0, hour_to_add = 0)

```

Comment:
Okay you see, it's not 12 because you can pass world.timeofday there and others.
worldtime2text is the widely used function for game time so it gets the config call to hour_start